show/main.py

```diff
@@ -2806,5 +2806,26 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+# show whether the auto-restart feature for containers or a specific container
+# is enabled or disabled.
+
+@cli.command('autorestart')
+@click.option('-c', '--container-name', required=False)
```

Comment:
No need for a flag. You can just add an optional parameter, which is more in line with our other SONiC commands. If the optional parameter is present (e.g., `show container feature autorestart swss`), limit your output to only that container. If it is not present (e.g., `show container feature autorestart`), display the status of all containers. Take a look through this file, there are many examples of this style.